dev/templates/all-items.html

```diff
@@ -0,0 +1,10 @@
+<table class="table table-striped table-condensed">
+  <thead>
+    <tr><th>name</th><th>brand</th><th>price</th></tr>
+  </thead>
+  <tbody>
+  {{#hits}}
+    <tr><td><a href="#hit-{{objectID}}">{{name}}</a></td><td>{{brand}}</td><td>{{price}}</td></tr>
```

Comment:
Well, it is doable with a small `Hogan` helper I believe. I can help you with that if you want.